pkg/virt-handler/device-manager/device_controller_test.go

```diff
@@ -211,5 +217,26 @@ var _ = Describe("Device Controller", func() {
 				return exists1 || exists2
 			}, 5*time.Second).Should(BeFalse())
 		})
+
+		It("Should not remove permanent device plugins if permittedHostDevices is removed from the CR", func() {
+			emptyConfigMap, _, _ := testutils.NewFakeClusterConfigUsingKVConfig(&v1.KubeVirtConfiguration{})
+			Expect(emptyConfigMap.GetPermittedHostDevices()).To(BeNil())
+
+			initialDevices := []Device{plugin1, plugin2}
```

Comment:
`initialDevices` -> `permanent` ?

Reply:
I have changed it to `permanentPlugins`, since that is the same name the actual device controller uses.